README.md

```diff
@@ -183,6 +184,74 @@
 
 <hr>
 
+<h3 id="camera-name-and-namespace">

```

Comment:
Please remember we will have to notify this change.
To validation to get ready for it, and other internal users as well
What will happen to users that doesn't change there launch file?
Maybe we need a red notice here about this change?
Let's speak and see how we can make a smooth transition as possible.

Reply:
I can use these kinds of notifications in the top of the README page
https://github.com/orgs/community/discussions/16925
We can tell validation and internal users about this before we merge this into SC branches.